administrator/components/com_cookiemanager/cookiemanager.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" method="upgrade">
+	<name>com_cookiemanager</name>
+	<author>Joomla! Project</author>
+	<creationDate>June 2021</creationDate>
+	<copyright>(C) 2021 Open Source Matters, Inc.</copyright>
+	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+	<authorEmail>admin@joomla.org</authorEmail>
+	<authorUrl>www.joomla.org</authorUrl>
+	<version>4.0.0</version>
```

Comment:
Version will be adapted

Reply:
version will be 1.0.0

Reply:
I didn't understand @chmst what you want to say.

Reply:
IIRC in one PR, Brian talk about this version adaption. I don't know how I will do it and I unable to find that PR.
@chmst I think you are talking about that.